Ops handbook tools (#832)
publiccodenet/about: activities/office-management/index.md

type: Index
---

> # Office and building management

Reviewer: Is this file change the same as PR #831?

Keizersgracht 617
1017 DS Amsterdam

> We have an anti-squat contract (bruikleenovereenkomst) with: K&O Group who manages the property on behalf of the owner, Hansainvest GmbH. The contract ends on 31 March 2021 with the possibility of extending it on a month-to-month basis until a new renter is found.

Reviewer 2: ```suggestion
We have an anti-squat contract (bruikleenovereenkomst) with K&O Group, which manages the property on behalf of the owner, Hansainvest GmbH. The contract ends on 31 March 2021 with the possibility of extending it on a month-to-month basis until the building is rented again.
```

Reviewer 2: The colon isn't necessary. K&O Group isn't a person.
